provision.yml

```diff
@@ -6,4 +6,11 @@
     - role: common
     - role: wordpress
 
+    - role: nginx
+      vhosts:
+        - name: staging.stockholmsaltspa.se
+          local_port: 8080
```

Comment:
Local port isn't 8080 yet, but you could quite easily make it so (and **should**) if you pass it into the role that creates the docker-compose file. :)